packages/storage/src/reference.ts

```diff
@@ -201,6 +203,34 @@ export class Reference {
     });
   }
 
+  /**
+   * List items and folders (prefixes) within this directory.
+   *
+   * @param {!options.maxResults} If set, limits the total number of prefixes
```

Comment:
Drop just the curly braces here to.

If you look at this in GitHub, you can actually see that the `If` is highlighted differently. The format is:

```
@param {type} argName Bla bla bla
```

Since this is TypeScript, you can drop `{type}`.